core/kernel/tainted_spec.rb

```diff
@@ -2,7 +2,7 @@
 require_relative 'fixtures/classes'
 
 describe "Kernel#tainted?" do
-  ruby_version_is ""..."3.0" do
+  ruby_version_is ""..."3.1" do
```

Comment:
Does `Object#tainted?` work in Ruby 3.1? `""..."3.1"` excludes the end boundary.
```suggestion
  ruby_version_is ""..."3.2" do
```